.github/workflows/e2e.yml

```diff
@@ -0,0 +1,275 @@
+      CONTAINER_SERVER: playwright_tests_server
+      IMAGE_CALLS_OFFLOADER: mattermost/calls-offloader:v0.8.0
+      IMAGE_CALLS_RECORDER: mattermost/calls-recorder:v0.7.1
+      IMAGE_CALLS_TRANSCRIBER: mattermost/calls-transcriber:v0.2.1
+      IMAGE_SERVER: mattermostdevelopment/mattermost-enterprise-edition:master
+      IMAGE_CURL: curlimages/curl:8.7.1
+      CI_NODE_INDEX: ${{ matrix.run_id }}
+      CI_NODE_TOTAL: ${{ needs.generate-matrix.outputs.parallelism }}
+    strategy:
+      matrix: ${{ fromJson(needs.generate-matrix.outputs.parallelism-matrix) }}
+    steps:
+      - name: e2e/checkout-mattermost-plugin-calls-repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: e2e/checkout-mattermost-repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          repository: mattermost/mattermost
+          path: mattermost
+
+      - name: e2e/setup-docker-buildx
+        uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # v3.3.0
+
+      - name: e2e/setup-go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          go-version-file: mattermost/server/go.mod
+
+      - name: e2e/generate-configuration
+        working-directory: ./mattermost/server/scripts/config_generator
+        env:
+          OUTPUT_CONFIG: ${{ github.workspace }}/config/config.json
+        run: |
+          mkdir -p ${{ github.workspace }}/config
+          go run main.go
+
+      - name: e2e/patch-generated-configuration
+        env:
+          OUTPUT_CONFIG: ${{ github.workspace }}/config/config.json
+        run: |
+          cp ${OUTPUT_CONFIG} ${OUTPUT_CONFIG}.orig
+          jq -f ${{ github.workspace }}/e2e/overlay-config.jq ${OUTPUT_CONFIG}.orig > ${OUTPUT_CONFIG}
+          diff ${OUTPUT_CONFIG}.orig ${OUTPUT_CONFIG} || true
+
+      - name: e2e/download-mattermost-plugin-calls-package
+        uses: actions/download-artifact@c850b930e6ba138125429b7e5c93fc707a7f8427 # v4.1.4
+        with:
+          name: mattermost-plugin-calls-package
+          path: dist
+
+      - name: e2e/prepare-server
+        env:
+          DOCKER_CLIENT_TIMEOUT: 120
+          COMPOSE_HTTP_TIMEOUT: 120
+          ## Should relative to mattermost/server/build/
+          DOCKER_COMPOSE_FILE: gitlab-dc.postgres.yml
+          DOCKER_COMPOSE_TEST_DATA: ../tests/test-data.ldif
+        run: |
+          mkdir -p ${{ github.workspace }}/logs
+          mkdir -p ${{ github.workspace }}/config
+          mkdir -p ${{ github.workspace }}/dotenv
+
+          ## Navigate to mattermost server build folder to spawn the environment
+          cd mattermost/server/build/
+          ${{ github.workspace }}/e2e/scripts/prepare-server.sh
+          cd -
+
+          ## Add extra environment variables for mattermost server
+          echo "MM_LICENSE=${{ secrets.MM_PLUGIN_CALLS_TEST_LICENSE }}" >> dotenv/app.private.env
```

Comment:
For my own understanding, where are these stored? Do we have a private repo somewhere?

Reply:
All our secrets are stored as organisation secrets. Unfortunately you do not have access there . 
Once we migrate to our secret management solution everything will be easier to handle